Gemfile

```diff
@@ -47,6 +48,12 @@ gem 'jquery-rails'
 # Vsphere access
 gem 'rbvmomi'
 
+# Mina for deployment
+gem 'mina', require: false
+gem 'mina-puma', require: false,  github: 'untitledkingdom/mina-puma'
```

Comment:
Links to the documentation of these projects / a sentence on what they do would be nice.